addons/pos_event/static/src/app/components/popup/event_configurator_popup/event_configurator_popup.js

```diff
@@ -95,8 +95,16 @@ export class EventConfiguratorPopup extends Component {
     ticketIsAvailable(ticket) {
         const dateTimeNow = DateTime.now();
         const bookedTicket = this.getOrderAlreadyBooked(ticket) + this.state[ticket.id].qty;
-        const eventAvailable =
-            !ticket.event_id?.seats_limited || bookedTicket <= ticket.event_id.seats_available;
+        let seatsAvailable = ticket.event_id.seats_available;
+        if (ticket.event_id?.is_multi_slots) {
+            // Filtered using the model pos domain (= no future slots)
+            const futureSlotTickets = ticket.slot_ticket_ids
+                .map((st) => st.slot_id?.id)
+                .filter((id) => id);
+            seatsAvailable = Math.max(...futureSlotTickets.map((st) => st.seats_available));
```

Comment:
And seats_available will account for both the slot_ticket and the ticket maximas, right? (I think so, just double checking here and then)